.github/workflows/ci.yml

```diff
@@ -32,20 +32,19 @@ jobs:
     - name: build and test
       run: |
         echo $TZ
-        go test -mod=vendor -timeout=60s -covermode=count -coverprofile=$GITHUB_WORKSPACE/profile.cov_tmp ./...
-        cat $GITHUB_WORKSPACE/profile.cov_tmp | grep -v "_mock.go" > $GITHUB_WORKSPACE/profile.cov
```

Comment:
this change losing ignore for generated files ( "_mock.go"). If we don't have any it is fine, but if we do - we want to exclude them from the coverage

Reply:
Don't have them in this project, that's the reason for dropping it.